src/sentry/auth/superuser.py

```diff
@@ -82,6 +96,9 @@ def __init__(self, request, allowed_ips=UNSET, org_id=UNSET, current_datetime=No
 
     @property
     def is_active(self):
+        org = getattr(self.request, "organization", None)
+        if org and org.id != self.org_id:
```

Comment:
when would the org not exist? and can we add a test case for that if so?

Reply:
I've noticed that this will be hit on everything that relies on the superuser class. since is_active_super() is called quite a bunch. if i put a breakpoint here, its hit even on login, during that time there isn't an org in the request, so thats when it doesn't exist. ill add a test case for this